docs/_posts/HashamUlHaq/2021-11-26-ner_biomarker_en.md

```diff
@@ -0,0 +1,176 @@
+results = model.transform(spark.createDataFrame([["Here , we report the first case of an intraductal tubulopapillary neoplasm of the pancreas with clear cell morphology . Immunohistochemistry revealed positivity for Pan-CK , CK7 , CK8/18 , MUC1 , MUC6 , carbonic anhydrase IX , CD10 , EMA , β-catenin and e-cadherin "]], ["text"]))
+
+```
+```scala
+...
+val embeddings_clinical = WordEmbeddingsModel.pretrained("embeddings_clinical", "en", "clinical/models")
+   .setInputCols(["sentence", "token"])
+   .setOutputCol("embeddings")
+
+val ner = MedicalNerModel.pretrained("ner_biomarker", "en", "clinical/models") 
+  .setInputCols("sentence", "token", "embeddings")
+  .setOutputCol("ner")
+...
+val pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, embeddings_clinical, ner, ner_converter))
+val data = Seq("Here , we report the first case of an intraductal tubulopapillary neoplasm of the pancreas with clear cell morphology . Immunohistochemistry revealed positivity for Pan-CK , CK7 , CK8/18 , MUC1 , MUC6 , carbonic anhydrase IX , CD10 , EMA , β-catenin and e-cadherin ").toDF("text")
+val result = pipeline.fit(data).transform(data)
+```
+</div>
+
+## Results
+
+```bash
+|    | ner_chunk                | entity                |   confidence |
+|---:|:-------------------------|:----------------------|-------------:|
+|  0 | intraductal              | CancerModifier        |     0.9934   |
+|  1 | tubulopapillary          | CancerModifier        |     0.6403   |
+|  2 | neoplasm of the pancreas | CancerDx              |     0.758825 |
+|  3 | clear cell               | CancerModifier        |     0.9633   |
+|  4 | Immunohistochemistry     | Test                  |     0.9534   |
+|  5 | positivity               | Biomarker_Measurement |     0.8795   |
+|  6 | Pan-CK                   | Biomarker             |     0.9975   |
+|  7 | CK7                      | Biomarker             |     0.9975   |
+|  8 | CK8/18                   | Biomarker             |     0.9987   |
+|  9 | MUC1                     | Biomarker             |     0.9967   |
+| 10 | MUC6                     | Biomarker             |     0.9972   |
+| 11 | carbonic anhydrase IX    | Biomarker             |     0.937567 |
+| 12 | CD10                     | Biomarker             |     0.9974   |
+| 13 | EMA                      | Biomarker             |     0.9899   |
+| 14 | β-catenin                | Biomarker             |     0.8059   |
+| 15 | e-cadherin               | Biomarker             |     0.9806   |
+
+```
+
+{:.model-param}
+## Model Information
+
+{:.table-model}
+|---|---|
+|Model Name:|ner_biomarker|
+|Compatibility:|Spark NLP for Healthcare 3.3.3+|
+|License:|Licensed|
+|Edition:|Official|
+|Input Labels:|[sentence, token, embeddings]|
+|Output Labels:|[ner]|
+|Language:|en|
+
+## Data Source
+
+Trained on data sampled from Mimic-III, and annotated in-house.
+
+## Benchmarking
+
+```bash
```

Comment:
This has a bad renderization. Format it.
![image](https://user-images.githubusercontent.com/36634572/144027020-00f08d82-8e67-4007-8f99-1f9302867b67.png)